services/ffc-demo/ffc-demo-calculation-service/helm-repository.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
```

Comment:
Do we need to add the HelmRepository to each service?

Reply:
It is just to provide the granularity to each service to reference the repository

Reply:
As discussed, we going to refine it later.